picard/coverart/image.py

```diff
@@ -228,7 +228,6 @@ def __repr__(self):
         return "%s(%s)" % (self.__class__.__name__, ", ".join(self._repr()))
 
     def _str(self):
-        yield 'Image'
```

Comment:
Not sure what the intention is, but having as string representation just "from /some/file/name" seems odd as well. What about outputting `self.__class__.__name__` here as well?

Reply:
Done in [6e0054d](https://github.com/metabrainz/picard/pull/2319/commits/6e0054d2423a8d06d2d32d35beca82cf24bd5e04)